dojo/tools/openvas/xml_parser.py

```diff
@@ -27,7 +28,9 @@ def get_findings(self, filename, test):
                     title = title + "_" + finding.text
                     description.append(f"**Port**: {finding.text}")
                 if finding.tag == "nvt":
-                    description.append(f"**NVT**: {finding.text}")
+                    script_id = finding.get("oid") or finding.text
```

Comment:
This may have some implication for hash code generation since the description is used by default. Please remove this section, and keep the `script_id` to the `vuln_id_from_tool` field